webserver/shared/models/AssetUtxos.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * @example "asset1c43p68zwjezc7f6w4w9qkhkwv9ppwz0f7c3amw"
+ */
+export type Cip14Fingerprint = string;
+
+export type AssetUtxosRequest = {
+  range: { minSlot: number; maxSlot: number },
```

Comment:
Again, let's not use this pagination style. 
https://dcspark.github.io/carp/docs/webserver/pagination

Reply:
Yeah, but I can't start integrating it on paima if I do that now, since that will require changing how the sync works. So I my plan was to start setting it up with this and then adapt it, since on this side it's not really that much work to change it.